cranelift/codegen/src/scoped_hash_map.rs

```diff
@@ -4,12 +4,12 @@
 //! container that has a concept of scopes that can be entered and exited, such that
 //! values inserted while inside a scope aren't visible outside the scope.
 
+use crate::FxHashMap;
 use core::hash::Hash;
-use rustc_hash::FxHashMap;
 use smallvec::{smallvec, SmallVec};
 
 #[cfg(not(feature = "std"))]
-use crate::fx::FxHasher;
+use rustc_hash::FxHasher;
```

Comment:
It doesn't look like `FxHasher` or `Hasher` are used anywhere in this module, and they aren't exported -- can we remove them instead?